api/version.go

```diff
@@ -0,0 +1,12 @@
+package api
+
+var (
+	// TODO: This goes away when control-plane-operator becomes another component
+	// in the OCP payload.
+	HyperShiftImage = "registry.ci.openshift.org/hypershift/hypershift:latest"
+
+	// OCPReleaseImage is the latest compatible OCP release, used for defaulting.
+	// This can and should be baked into the binary through ldflags, although
+	// the version committed here should be kept up to date with major releases.
+	OCPReleaseImage = "quay.io/openshift-release-dev/ocp-release:4.7.0-fc.3-x86_64"
```

Comment:
I'm thinking that maybe there is a better place for this. Conceptually, the hypershift API should not be related to an OCPReleaseImage.

Reply:
The 2 places where it's used is in the cluster create command and the e2e. Could it be part of the cluster create command?

Reply:
What about an `openshift.io/hypershift/version` package?

Reply:
I guess that's better, but I was thinking of maybe just grabbing the latest stable one instead of having a static variable. For example, default to https://openshift-release.svc.ci.openshift.org/api/v1/releasestream/4-stable/latest

Reply:
We talked and decided the dynamic approach is better, which is now implemented.